packages/web-app-admin-settings/src/components/Spaces/SpacesList.vue

```diff
@@ -326,19 +327,19 @@ export default defineComponent({
       return formatRelativeDateFromJSDate(new Date(date), $language.current)
     }
     const getTotalQuota = (space: SpaceResource) => {
-      return `${space.spaceQuota.total * Math.pow(10, -9)} GB`
+      return filesize(space.spaceQuota.total)
```

Comment:
Could you use the `formatFileSize` helper from `web-pkg/src/helpers/filesize` instead? For two reasons:
1. makes sure that the correct localization is used
2. handles invalid values in a consistent way